front/README.md

```diff
@@ -76,10 +70,14 @@ For example, to add XYZ to the `package.json` file in the `front` folder while d
 Afterwards:
 
 1. stop docker-compose and run `docker compose up` again
-2. run `pnpm` in your local front folder (so that your tooling like VSCode also picks up the changes, like new TS types etc)
+2. run `pnpm i` in your local front folder (so that your tooling like VSCode also picks up the changes, like new TS types etc)
 
 (This advice has come from https://github.com/BretFisher/node-docker-good-defaults)
 
+## About Husky
+
+`husky` is a git hook tool that we use to format and lint staged files on pre-commits. In case you use a version manager tool (e.g. `nvm`, `asdf`), or run into trouble commiting your code, consult https://typicode.github.io/husky/how-to.html#node-version-managers-and-guis.
```

Comment:
to be able to run husky and lint-staged, we need to install node locally, correct?

Reply:
That's correct @HaGuesto .